### PR TITLE
feat: Added new Fields to App Table 

### PR DESCRIPTION
**Pull request by andy-t-wang**
Synced with Ian and Michael today and they wanted some more fields in the app table. Added them as well as a trigger to check user inputted urls are correct. No option to currently include them on the form. Will add that with the pending_app_table front end change

### CODE REVIEW

File: hasura/migrations/default/1705005696536_squashed/up.sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
+alter table "public"."app"
+add column "showcase_img_urls" jsonb null;
```
Comment (pdtfh) [inline hasura/migrations/default/1705005696536_squashed/up.sql:2]: not a fan of having this as a JSON, will require different handling than the rest of the images, can we add this later?

File: hasura/metadata/databases/default/tables/public_app.yaml [hunk 2]
```diff
@@ -19,6 +19,11 @@ computed_fields:
       function:
         name: get_app_is_verified
         schema: public
+  - name: show_details_page
+    definition:
```
Comment (pdtfh) [inline hasura/metadata/databases/default/tables/public_app.yaml:23]: what is this doing?

Comment (andy-t-wang) [reply hasura/metadata/databases/default/tables/public_app.yaml:23]: A field that is used in the app to decide to open the details page or just go directly to the link.

File: hasura/migrations/default/1705005696536_squashed/up.sql (hunk 1 above)
Comment (andy-t-wang) [reply hasura/migrations/default/1705005696536_squashed/up.sql:2]: ok I will make this text [] instead. And we can have the user store the paths but I can just add a validation on the db that the inputs can only be the fixed paths we predetermined

**Comment by andy-t-wang** [comment]
We are changing the structure to a single metadata table to manage this. See updated specs WDP25

**Pull request closed without merging**